process/smartContract/testUtils.go

```diff
@@ -105,3 +105,13 @@ func GetAllSCRs(scProcessorAsInterface interface{}) []data.TransactionHandler {
 
 	return scrProvider.GetIntermediateTransactions()
 }
+
+// CleanGasRefunded cleans the gas computation handler
+func CleanGasRefunded(scProcessorAsInterface interface{}) {
```

Comment:
Totally disagree to use this type of code even in tests. It breaks all rules regarding clean code. Let's find some other way to do the same job. I know this file contains all sort of functions but they are terrible horrible hacks and should be removed.